src/objects.js

```diff
@@ -1,41 +1,64 @@
 const createPerson = (name, age) => {
-  // your code here
+  return { name, age };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]) return true;
```

Comment:
Nice, you could also use `return Boolean(object[property])` or Object.hasOwnProperty() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty